x-pack/test/security_functional/tests/security_check/unsecure_cluster_alert.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const find = getService('find');
+  const PageObjects = getPageObjects(['common']);
+  const esArchiver = getService('esArchiver');
+
+  describe('Unsecure Cluster Alert', function () {
+    before(async () => {
+      await esArchiver.emptyKibanaIndex();
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
```

Comment:
__note:__ sorry for not mentioning this earlier, but we're trying to get rid of all es archives as they are quite expensive (they cause SO migrations). Can't we just create necessary conditions for the insecure cluster warnings using the Kibana\ES APIs directly?